dev/com.ibm.ws.security.authentication.filter_fat/.classpath

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/build.sharedResources/lib/junit/old/junit.jar">
+		<attributes>
+			<attribute name="bsn" value="../build.sharedResources/lib/junit/old/junit.jar"/>
+			<attribute name="type" value="EXTERNAL"/>
+			<attribute name="project" value="build.depScanner"/>
+			<attribute name="version" value="file"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="src" path="/fattest.simplicity"/>
```

Comment:
Same as above. Almost all, if not all these libraries should be pulled in from the bnd.bnd file. This is done by the fact you have the `aQute.bnd.classpath.container` classpath entry above.